ArgosIOS/Argos/Views/LandingPage/LandingView.swift

```diff
@@ -33,10 +34,14 @@ struct LandingView: View {
                         ArgosButton(title: "Historical", action: {
                             self.viewModel.dialogPresentation.show(content: .carousel(runs: props.runs, selectRun: self.viewModel.selectRun, isPresented: self.$viewModel.dialogPresentation.isPresented))
                         })
-                        
+    
                         ArgosButton(title: "More Details", action: {
                             self.viewModel.onMoreDetailsClicked()
                         })
+                        
+                    }
+                    NavigationStack {
```

Comment:
if you look at the other buttons, youll see we dont use navigation links for navigation, just do what we do here, or create a ArgosNavigation hashable enum and then use Navigation(for: ) to do navigation. Dont use a navigation link

Reply:
aw i was misunderstanding the button navigation my bad. can i ask why we arent using nav links? i dont know much about swift

Reply:
for scalability its better to use enumerations for navigation paths. Once we add tabs were going to have to do it this way in order to have features like pressing a tab brings you back to the root navigation. Navigation links are good for when you have like a table view and want to send information to the next view based on what item is pressed but since we dont care about any of that we can use enumerations very easily

Reply:
ahh i see gotcha